Code/Framework/AzToolsFramework/AzToolsFramework/Entity/PrefabEditorEntityOwnershipService.cpp

```diff
@@ -359,6 +359,16 @@ namespace AzToolsFramework
         return AZStd::nullopt;
     }
 
+    Prefab::TemplateId PrefabEditorEntityOwnershipService::GetRootPrefabTemplateId()
+    {
+        AZ_Assert(m_rootInstance, "A valid root prefab instance couldn't be found in PrefabEditorEntityOwnershipService.");
+        if (m_rootInstance)
+        {
+            return m_rootInstance->GetTemplateId();
+        }
+        return Prefab::InvalidTemplateId;
```

Comment:
tiny suggestion to simplify this code:
```suggestion
        return m_rootInstance ? m_rootInstance->GetTemplateId() : Prefab::InvalidTemplateId;
```